libc/src/__support/CPP/bit.h

```diff
@@ -268,6 +268,65 @@ SPECIALIZE_FLZ(first_leading_zero, unsigned long long, __builtin_clzll)
 
 #undef SPECIALIZE_FLZ
 
+#define SPECIALIZE_FLO(NAME, TYPE, BUILTIN)                                    \
```

Comment:
I think there is no need for the specializations here because `countr_zero` and `countl_zero` themselves are already specialized.

Reply:
sure, done in https://github.com/llvm/llvm-project/pull/81502/commits/ce727813aad9428ac2803f77bf143637ffa7a6e7 

Note: this PR is stacked on top of #81502. #81502 should land first, then this PR.

Reply:
Also sent https://github.com/llvm/llvm-project/pull/81678 which is inspired by this suggestion.